doc/suiterc.tex

```diff
@@ -1331,14 +1350,14 @@ \subsection{[runtime]}
 \subparagraph[shell]{[runtime] \textrightarrow [[\_\_NAME\_\_]] \textrightarrow [[[job]]] \textrightarrow shell}
 \label{JobSubShell}
 
-This is the shell used to interpret the job script submitted by the suite
-daemon when a task is ready to run.  {\em It has no bearing on the shell used
-in task implementations.} Scripting and suite environment
-variable assignment expressions must be valid for this shell. The
-latter is currently hardwired into cylc as
-\lstinline@export item=value@ - valid for both bash and ksh
-because \lstinline=value= is entirely user-defined - but cylc would have
-to be modified slightly to allow use of the C shell.
+Location of the command used to interpret the job script submitted by the
+suite daemon when a task is ready to run. This can be set to the location of
+\lstinline=bash= in the job host if the shell is not installed in the standard
+location.
+{\em Note: It has no bearing on any sub-shells that may be called by the job script.}
+
+Setting this to the path of a ksh93 interpretter is supported but not recommended.
```

Comment:
Shall we give a deprecation warning for ksh93 (here, and print one if detected on file parsing) - so we can drop support sometime in the future?